src/screens/FeedScreen/ContentPoll.js

```diff
@@ -309,15 +310,15 @@ const stylesComponent = (pollLength) =>
       alignSelf: 'center'
     },
     seeresultstext: {
-      color: COLORS.holyTosca,
+      color: colors.holytosca,
       fontFamily: fonts.inter[500],
       fontSize: normalizeFontSize(14)
     },
     textMedia: {
       fontFamily: fonts.inter[400],
       fontWeight: 'normal',
       fontSize: normalizeFontSize(FONT_SIZE_MEDIA),
-      color: COLORS.black,
+      color: colors.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated from `COLORS.black` to `colors.black`. Make sure that `colors.black` is defined and is the intended color.
```diff
- color: COLORS.black
+ color: colors.black
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->